### ISSUE TITLE
Missing license

### ISSUE THREAD

**Issue by fieldOfView** [FormerLurker/ArcWelderLib#17]
This project seems to be missing a license. Is the license the same AGPL as the ArcWelderPlugin?

**Comment by FormerLurker** [comment]
Hi @fieldOfView!  This slipped through the cracks, sorry.  I will update the license as soon as I have a chance.  I intend to use AGPL2.

**Comment by atsju** [comment]
Hello, this is more or less related but I suppose that you will get more and more issues here doe to the arc welder plugin beeing integrated to Cura. Consider adding an issue_template.md file to get good quality tickets.

**Comment by FormerLurker** [comment]
@atsju, good idea.  I have done that for Octolapse, but pretty much everyone just ignores it.  However,  there is a really awesome feature coming soon that allows forms within the issue template, so I may wait for that.  I am also planning to add a bot at some point.

Just FYI, I wrote the core ArcWelder algorithm, the console app, and the OctoPrint plugin, but @fieldOfView wrote the cura plugin.  It's a difficult setup regarding who gets what issue, but we pretty much refer to each other's repositories as necessary, depending on the issue.

Oh, @fieldOfView, I really need to post the license, lol!  So many things..

**Comment by atsju** [comment]
@FormerLurker it's really easy to add a licence. Just clic "add file" in Github interface and name it licence. A button to use one of the templates will appear.
![image](https://user-images.githubusercontent.com/4628382/108618598-b2f2d900-741f-11eb-9651-d592a9598ce7.png)

The interface is really good with a simplified view of permissions/limitations/conditions. 

![image](https://user-images.githubusercontent.com/4628382/108618653-1ed54180-7420-11eb-8272-23628fd95f8b.png)

I was about to do a PR for you but you mentioned "AGPL2" and the template propose only AGPL3 or GPL2. So I suppose it's better to give you the tip than to propose the PR.

**Comment by FormerLurker** [comment]
Yeah @atsju, I'll have to mirror the arc welder plugin license.  Thanks for the tips!
